src/components/commons/NavHeaderContent.tsx

```diff
@@ -15,16 +17,32 @@ const NavHeaderContent = () => {
 export default NavHeaderContent;
 
 const NavHeaderWrapper = styled.div`
```

Comment:
여기서 $isScroll 사용할 때 제네릭으로 $isScroll type 지정해두면 좋을 것 같아요
```suggestion
const NavHeaderWrapper = styled.div<{$isScroll: boolean}>`
```

Reply:
맞아요 이거 안해두니까 다 에러뜨드라구요!!! 한군데만 해두고 다른데를 안해놔서 다 적용시키겠습니다!